CONTRIBUTING.md

```diff
@@ -0,0 +1,156 @@
+# How to contribute to EquinoxJS
+
+We would love for you to contribute to EquinoxJS and help make it even better
+than it is today! As a contributor, here are the guidelines we would like you
+to follow:
+
+## <a href="question"></a> Got a question or problem?
+
+**Do not open issues for general support questions. Github issues are reserved
+for bug reports and feature requests**. To get your questions answered please
+head over to the [DDD-CQRS-ES discord][discord].
+
+## <a href="bug"></a> Found a bug?
+
+If you find a bug in the source code, you can help us by
+[submitting an issue](#submit-issue) to our [GitHub Repository][github]. Even better, you can
+[submit a Pull Request](#submit-pr) with a fix.
+
+## <a name="feature"></a> Missing a Feature?
+
+You can _request_ a new feature by [submitting an issue](#submit-issue) to our GitHub
+Repository. If you would like to _implement_ a new feature, please submit an issue with
+a proposal for your work first, to be sure that we can use it.
+Please consider what kind of change it is:
+
+- For a **Major Feature**, first open an issue and outline your proposal so
+  that it can be discussed. This will also allow us to better coordinate our
+  efforts, prevent duplication of work, and help you to craft the change so
+  that it is successfully accepted into the project. For your issue name,
+  please prefix your proposal with `[discussion]`, for example "[discussion]:
```

Comment:
Perhaps can this be done in GH discussions? I do agree it's better for to/fro to happen outside of GH issues prior to actually creating a Feature Request for something big and/or requiring fleshing out. Having said that, major features are relatively rare so majobe this is fine.

Reply:
I haven't used GH discussions, have you had success with it?

Reply:
works fine - has notifications, differnt tab to issues, has a different emphasis
I guess its a Q of whether you imagine it being used enough
https://github.com/xunit/xunit/discussions